### DISCUSSION TITLE
Problem with Pipe Flow case in Incompact3D-v4.1

### DISCUSSION THREAD

**Discussion by flowDomainSmruti** [xcompact3d/Incompact3d#272] [Q&A] [answered]
Hi Everyone,

I have recently started using the **Incompact3D-v4.1** & as a part of the validation exercise, I wanted to run the pipe flow test case, however, while running it in the cluster, the **case stops abruptly after geometry creation** (log file attached). While running my case, I just used the same **input_laminar.i3d** file with minor changes for certain parameters (input file attched) which I don't think should affect the overall simulation.

Could the admin please look into it?

Regards.

[aaaoutput.1102638.un05.log](https://github.com/user-attachments/files/15774697/aaaoutput.1102638.un05.log)
[input.txt](https://github.com/user-attachments/files/15774742/input.txt)

**Comment by slaizet** [comment]
Hi,
Please use the other two input files for pipe flows and iibm=2.
Best,
Sylvain

**Reply by flowDomainSmruti** [reply]
As suggested, I have tried running the alternate input file, namely, input_DNS_Re5300_HR.i3d file for simulating the pipe flow case which was also there in the same folder. But, still the simulation stops immediately after generating the geometry (log file attached).

I have chosen iibm=2, but still the simulation stops. I believe there could be a problem with this version of the code. Could you please look into it?

[aaaoutput.1110274.un05.log](https://github.com/user-attachments/files/15965618/aaaoutput.1110274.un05.log)

**Comment by slaizet** [comment, marked as answer]
Hi,
I am able to run a pipe flow simulation with the latest version of the code (I used `input_DNS_Re1000_LR.i3d`). I used the GNU compiler. I would suggest you use the latest version of the code.
Best,

**Comment by Madhurjya814** [comment]
Hi,
With the latest version of the code when I tried to run the Re1000 and Re5300 for pipeflow case it showed Fortran runtime error in the line 240 of the parameters.f90 file. Can you help me why this error is coming? Thanks.
Regards
Madhurjya

**Reply by mathrack** [reply]
Hello,

I think line 239 of parameters.f90 (https://github.com/xcompact3d/Incompact3d/blob/master/src/parameters.f90#L239) should be

`if (itype.eq.itype_ptbl) then` or `if (itype.eq.itype_tbl) then`

Thank you for the information. You can open a bug report. This bug was introduced in https://github.com/xcompact3d/Incompact3d/pull/286.

**Reply by Madhurjya814** [reply]
Hello, 
I have tried the new line you have given. But now I am getting a different error. I am attaching the image. Thank you.
![Screenshot 2024-10-19 165627](https://github.com/user-attachments/assets/69074775-bf48-4527-8563-8644bbe1dd18)

**Reply by slaizet** [reply]
I will have a look into this again but please can you provide me with your input file? Also which version of the code did you use?
Cheers,
Sylvain
